### PR TITLE
Member can see crops they have requested

### PR DESCRIPTION
**Pull request by Br3nda**
This is for  #1039

Needs specs

### CODE REVIEW

**Comment by Br3nda** [comment]
spec added

File: config/locales/en.yml [hunk 1]
```diff
@@ -160,37 +164,37 @@ en:
...
   seeds:
-    string: "%{crop} seeds belonging to %{owner}"
     form:
       trade_help: Are you interested in trading or swapping seeds with other %{site_name} members?  If you list your seeds as available for trade, other members can contact you to request seeds.  You can list any conditions or other information in the description, above.
     index:
       title:
         crop_seeds: Everyone's %{crop} seeds
         default: Everyone's seeds
         owner_seeds: "%{owner} seeds"
+    string: "%{crop} seeds belonging to %{owner}"
```
Comment (Br3nda) [inline config/locales/en.yml]: i18n-tasks is moving these - i reckon we go with it

File: app/views/crops/requested.haml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+- content_for :title, t('crops.requested.title')
+- content_for :subtitle, t('crops.requested.subtitle', crops_size: @crops.size)
+
+%p Crops you have requested
```
Comment (CloCkWeRX) [inline app/views/crops/requested.haml]: Worth internationalising? (everything else is)

Comment (pozorvlak) [inline app/views/crops/requested.haml]: +1

**Pull request merged**